src/systems/filecoin_mining/storage_mining/constants.go

```diff
@@ -11,3 +14,7 @@ const PROVING_PERIOD = block.ChainEpoch(2)                    // placeholder
 const SURPRISE_CHALLENGE_FREQUENCY = 2 // placeholder
 // how far into their PP does a miner get their first challenge
 const SUPRISE_NO_CHALLENGE_PERIOD = PROVING_PERIOD / SURPRISE_CHALLENGE_FREQUENCY
+
+// FIL deposit per sector precommit in Interactive PoRep
+// refunded after ProveCommit but burned if PreCommit expires
+const PRECOMMIT_DEPOSIT = actor.TokenAmount(0) // placeholder
```

Comment:
Should this be a function of sector size, so a deposit per byte or per GiB?

Reply:
Not sure, but think it's a good suggestion, I can add that for now, simple change to add/remove later.

Reply:
cc @jzimmerman @nicola